client/web/compose/src/components/Namespaces/NamespaceSidebar.vue

```diff
@@ -319,6 +305,26 @@ export default {
   },
 
   methods: {
+    moduleWrap (module) {
+      const routeName = this.$route.name
+      const currentPageType = routeName.endsWith('record.list') ? 'record.list' : 'edit'
```

Comment:
you shouldd check if you're on edit, to make the record view/edit/create, behave correctly